api/api.proto

```diff
@@ -728,6 +728,8 @@ message StorageNode {
   string hostname = 15;
   // User defined labels for the node
   map<string, string> node_labels = 16;
+  // Name of the node
+  string name = 17;
```

Comment:
See `api.go`. So this should be `scheduler_host_name`. The comments on this file are directly used for the documentation website.